src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  if(days<3){
+    return days*RENT_COST;
+  } else if(days < 7){
+    return days*RENT_COST-20;
```

Comment:
The comparison `days < 7` is correct, but the literal `7` should be replaced with a constant that explains its meaning. Also, the discount value `20` should be a named constant.